benchmarks/asv.conf.json

```diff
@@ -43,7 +51,12 @@
     // environments in.  If not provided, defaults to "env"
     "env_dir": "env",
 
-    "environment_type": "virtualenv",
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    // "environment_type": "virtualenv",
```

Comment:
Looks helpful.

Reply:
Agreed. I don't understand why asv devs would want to get rid of autodetection. Assuming users use a particular type of environment is very annoying for those users who are using a different env management method.